### PR TITLE
#8096 Remove Apache Commons usages

### LINKED ISSUE #8096: Remove Apache Commons usages

### PR DESCRIPTION
**Pull request by timo0**
Fix #8096

### CODE REVIEW

File: platform-sdk/swirlds-base/src/main/java/com/swirlds/base/utility/StringUtils.java
Comment (hendrikebbers) [inline platform-sdk/swirlds-base/src/main/java/com/swirlds/base/utility/StringUtils.java:39]: Since some versions `java.lang.String` provides `isBlank()`. Please use that one instead of a custom method.

File: platform-sdk/swirlds-common/src/main/java/com/swirlds/common/exceptions/NotImplementedException.java
Comment (hendrikebbers) [inline platform-sdk/swirlds-common/src/main/java/com/swirlds/common/exceptions/NotImplementedException.java:19]: Javadoc

File: platform-sdk/swirlds-base/src/main/java/com/swirlds/base/utility/ExceptionUtils.java
Comment (hendrikebbers) [inline platform-sdk/swirlds-base/src/main/java/com/swirlds/base/utility/ExceptionUtils.java:24]: This is only used in the browser UI. Please move it to swirlds-common or if possible platform-gui or platform-core.

File: platform-sdk/swirlds-base/src/main/java/com/swirlds/base/utility/StringUtils.java
Comment (hendrikebbers) [inline platform-sdk/swirlds-base/src/main/java/com/swirlds/base/utility/StringUtils.java:68]: Please move that class to swirlds-common. The only method that it contains is quite specific / unusual and therefore I do not want to have it in base ;)

File: platform-sdk/swirlds-common/src/main/java/com/swirlds/common/utility/CommonUtils.java
Comment (cody-littley) [inline platform-sdk/swirlds-common/src/main/java/com/swirlds/common/utility/CommonUtils.java:366]: This method should be deprecated in favor of `ByteUtils.intToByteArray()`.

File: platform-sdk/swirlds-common/src/main/java/com/swirlds/common/utility/CommonUtils.java
Comment (cody-littley) [inline platform-sdk/swirlds-common/src/main/java/com/swirlds/common/utility/CommonUtils.java:429]: There is already a platform utility for creating human readable strings out of things like bytes.

`byteCountToDisplaySize(1234)` can be replaced with `new UnitFormatter(1234, UNIT_BYTES).render()`. See `UnitFormatterTests.java` for further examples.

File: platform-sdk/swirlds-common/src/main/java/com/swirlds/common/utility/CommonUtils.java
Comment (cody-littley) [inline platform-sdk/swirlds-common/src/main/java/com/swirlds/common/utility/CommonUtils.java:446]: There is a platform utility for this. 

`leftPad("this is my string", 100, ' ')` is equivalent to `ALIGNED_LEFT.pad("this is my string", ' ', 100)`. See `HorizontalAlignmentTests.java` for more examples.

File: platform-sdk/swirlds-common/src/main/java/com/swirlds/common/utility/ExceptionUtils.java
Comment (cody-littley) [inline platform-sdk/swirlds-common/src/main/java/com/swirlds/common/utility/ExceptionUtils.java:36]: There is a platform utility already for this. Use `StackTrace.getStackTrace(t).toString()` for equivalent functionality.

**Review by imalygin — APPROVED** [review]

File: platform-sdk/swirlds-common/src/main/java/com/swirlds/common/utility/CommonUtils.java
Comment (cody-littley) [inline platform-sdk/swirlds-common/src/main/java/com/swirlds/common/utility/CommonUtils.java:415]: A unit formatter automatically chooses a good unit for the provided quantity, so no need to do that manually.

```suggestion
    public static String byteCountToDisplaySize(final long bytes) {
        return UNIT_BYTES.buildFormatter(bytes)
            .setDecimalPlaces(1)
            .render();
    }
```

**Review by cody-littley — APPROVED** [review]

**Review by hendrikebbers — APPROVED** [review]

**Review by hendrikebbers — DISMISSED** [review]

**Review by imalygin — DISMISSED** [review]

File: build-logic/src/main/kotlin/com.hedera.hashgraph.jpms-modules.gradle.kts
Comment (cody-littley) [inline build-logic/src/main/kotlin/com.hedera.hashgraph.jpms-modules.gradle.kts:1]: Please do not merge until the JRS panel is passing.

**Review by cody-littley — APPROVED** [review]

**Review by nathanklick — DISMISSED** [review]

**Comment by nathanklick** [comment]
@timo0 Approved; however, the failing JRS tests need to be fixed before merging.

**Review by qnswirlds — APPROVED** [review]

**Review by kimbor — APPROVED** [review]

**Review by hendrikebbers — APPROVED** [review]

**Review by hendrikebbers — APPROVED** [review]

**Review by cody-littley — APPROVED** [review]

**Review by imalygin — APPROVED** [review]

**Pull request merged**
